libkbfs/backpressure_disk_limiter.go

```diff
@@ -469,6 +480,52 @@ func (bdl *backpressureDiskLimiter) onBlocksDelete(
 	bdl.fileTracker.onBlocksDelete(blockFiles)
 }
 
+func (bdl *backpressureDiskLimiter) onDiskBlockCacheDelete(
+	ctx context.Context, blockBytes int64) {
+	if blockBytes == 0 {
+		return
+	}
+	bdl.lock.Lock()
+	defer bdl.lock.Unlock()
+	bdl.diskCacheByteTracker.onBlocksDelete(blockBytes)
+}
+
+func (bdl *backpressureDiskLimiter) beforeDiskBlockCachePut(
```

Comment:
I don't see a new `onDiskBlockCacheEnable(cacheBytes int64)` function. I think that's still necessary, as otherwise every time we restart we'd think the cache is empty.